core/src/components/breadcrumb/breadcrumb.vars.scss

```diff
@@ -7,7 +7,7 @@
 $breadcrumb-font-weight:         400 !default;
 
 /// @prop - Font size of the breadcrumb
-$breadcrumb-font-size:           16px !default;
+$breadcrumb-font-size:           1rem !default;
```

Comment:
Do we also want to support a minimum font-size in case this value changes in the future or do we want to deal with it then? We could just copy the values back-button uses.

Reply:
Good question! I don't have any strong feelings. iOS doesn't really have this UI pattern, so we can make a call. Any preference?

Reply:
I didn't test this out but does the `body` text have a minimum font size? Maybe we should match what that does? Or we could just make the minimum font size what it currently is (16px). I don't know what the most relatable component would be here.

Reply:
Yeah good thought -- the components on iOS all have different minimum font sizes, so not sure that would work for all components. For example, chip has a minimum font size of 11px or 13px or something similar: https://developer.apple.com/design/human-interface-guidelines/typography#Specifications

iOS also has some built-in minimum font sizes. It looks like the "Body" font size does not go smaller than 14px on iOS.

Reply:
Oh yeah I was thinking maybe just for breadcrumbs we match the body minimum since it could be placed in the body too.

Reply:
Fixed in https://github.com/ionic-team/ionic-framework/pull/27759/commits/7227630495891d1f0a7cf826e8323c072043021e